packages/coinjoin/src/utils/roundUtils.ts

```diff
@@ -138,24 +139,18 @@ export const findNearestDeadline = (rounds: Round[]) => {
 
 // iterate Status and find relevant round data
 export const getDataFromRounds = (rounds: Round[]) => {
-    const [coordinatorFeeRates, maxAllowedAmounts, minAllowedAmounts] = rounds.reduce(
-        ([previousRates, previousMax, previousMin], round) => {
-            const roundParameters = getRoundParameters(round);
-            if (roundParameters) {
-                previousRates.push(roundParameters.coordinationFeeRate.rate);
-                previousMax.push(roundParameters.allowedInputAmounts.max);
-                previousMin.push(roundParameters.allowedInputAmounts.min);
-            }
-            return [previousRates, previousMax, previousMin];
-        },
-        [[], [], []] as number[][],
-    );
+    const roundParameters = getRoundParameters(rounds[rounds.length - 1]);
```

Comment:
nit: perhaps we could start using [`rounds.at(-1)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)?

Reply:
Interestingly, this returns another type (`Round | undefined`) despite the result in JS being the same... anyway, changed in 51c768b46059ccd7eb060f3f50a7dda5ee935c0b.

Reply:
Well, if u think about it, that type is more correct 😅  When you enter a non-existent index in `[]` you get `undefined`